components/NavigationBar.js

```diff
@@ -19,11 +20,12 @@ export default function NavigationBar () {
   )
 }
 
-function NavigationLink ({ href, disabled, children }) {
+function NavigationLink({ href, disabled, children }) {
   return (
-    <span className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
-      <style jsx>{`
+    <Link to={children.toLowerCase()} smooth={true} duration={500} >
```

Comment:
Using a `Link` component from `react-scroll` means that the user cannot Cmd+Click to open the link in a new tab.